db/seeds.rb

```diff
@@ -24,6 +24,18 @@
   email_confirmed: true
 ).find_or_create_by!(email: "gem-requester@example.com")
 
+user_with_yubikey = User.create_with(
```

Comment:
I added a user with a mock yubikey to test the mailer preview. It isn't intended to be used by developers since they are unable to login (blocked by the MFA page with mock webauthn credentials).

Reply:
Actually nevermind, I'll initialize a webauthn credential in the mailer preview since it would be only used there.